### PR TITLE
LPS-154913 [Impedibug] [CP 2.0] Modal of confimation DXP Cloud and Analytics Cloud

### PR DESCRIPTION
**Pull request by thaynam**

### CODE REVIEW

**Review by jairmedeiros — APPROVED** [review]

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/I18n/en_US.json [hunk 1]
```diff
@@ -301,6 +303,7 @@
 	"virtual-cluster-x-nodes": "Virtual Cluster ({0} nodes)",
 	"we-ll-need-a-few-details-to-finish-building-your-analytics-cloud-workspace-s": "We'll need a few details to finish building your Analytics Cloud workspace(s).",
 	"we-ll-need-a-few-details-to-finish-building-your-dxp-environment": "We'll need a few details to finish building your DXP environment",
+	"we-ll-need-a-few-details-to-finish-building-your-dxp-environment-s": "We’ll need a few details to finish building your DXP environment(s).",
```
Comment (claricebrasil) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/I18n/en_US.json:306]: I believe it is better to leave the word in the plural (environments)

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/onboarding/pages/SuccessCloud/index.js [hunk 2]
```diff
@@ -9,26 +9,29 @@
...
-		helper:
-			'We’ll need a few details to finish building your DXP environment(s).',
-		paragraph: `Thank you for submitting this request! Your DXP Cloud project
-			will be provisioned in 2-3 business days. At that time, DXP
-			Cloud Administrators will receive several onboarding emails,
-			giving them access to all the DXP Cloud environments and tools
-			included in your subscription.`,
-		title: 'Set up DXP Cloud',
+		helper: i18n.translate(
+			'we-ll-need-a-few-details-to-finish-building-your-dxp-environment-s'
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/onboarding/pages/SuccessCloud/index.js:29]: Follow the suggestions of @claricebrasil, here is another example to add as plural :)

Comment (claricebrasil) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/onboarding/pages/SuccessCloud/index.js:29]: Better to leave environment in plural

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

**Review by claricebrasil — APPROVED** [review]
I made some suggestions according to Brian's past observations

**Review by jairmedeiros — APPROVED** [review]

**Comment by jairmedeiros** [comment]
ci:forward

**Pull request closed without merging**